package/src/renderer/Canvas.tsx

```diff
@@ -87,6 +87,32 @@ export const Canvas = forwardRef<SkiaView, CanvasProps>(
       () => skiaReconciler.createContainer(tree, 0, false, null),
       [tree]
     );
+    // Add subscription to values in all child properties
+    useEffect(() => {
```

Comment:
I was wondering about restricting animation values to be created only in the Skia renderer.
This would allow animation values to push their subscriptions instead of doing a pulling (here it might also be expensive and also we will do this traverse even when not needed).

Another benefit of that is that we could remove addListener() since useValueEffect could be implemented in the renderer()

Of course a huge problem with that is the onTouch that is declared outside. We could add a component (`<TouchHandler onTouch={onTouch}`) that would live in the renderer or have `enumChildren` to work at only one level (no depth)).  Not sure yet 🤔